src/js/claims-status/utils/helpers.js

```diff
@@ -243,3 +243,97 @@ export function getCompletedDate(claim) {
 export function getClaimType(claim) {
   return claim.attributes.claimType || 'Disability Compensation';
 }
+
+// TO DO: Replace made up properties and content with real versions once finalized.
+export const STATUS_CONTENT = {
```

Comment:
Can we rename this to clarify that this holds the types? Maybe `CURRENT_STATUS_TYPES`? Also, we might want to switch the keys and values around so we can use the snake_case string we get from the api as the key.

🤔  Are the current status types different than past event types, or do they share a common status type enum?

Reply:
I believe they're the same, so going with STATUS_TYPES. Based on what I've seen so far, each status has its place in the chronological appeal order, and can either be the current status, past status, or next status. Let's confirm with appeals team.